### DISCUSSION TITLE
Respawn.

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#81] [Ideas]
used a respawn feature from this https://github.com/KaylinOwO/Project-Apparatus, it works amazingly! turns myself invisible to my teammates but not the enemies and i can respawn as many times as i like. is it possible to replicate this? also it has a feature to carry everything in one arm even bodies xd.

**Comment by winstxnhdw** [comment]
I can take a look at it.

**Reply by winstxnhdw** [reply]
I tried that method before though. Are you sure it works if you aren't host?

**Reply by yujispin990** [reply]
so far i think it works, i could bring items everywhere, i could kill enemies and teammates.

**Reply by yujispin990** [reply]
how about you try his project out maybe there are differences that i dont know about?

**Reply by yujispin990** [reply]
i would record but my pc has some problems, the only issue is if you dont kill yourself before the ship completely leaves, the ship is bugged.
